.golangci.yml

```diff
@@ -109,37 +144,188 @@ linters-settings:
   gofumpt:
     extra-rules: true
   govet:
-    enable-all: true
-    disable:
+    disable-all: true
+    enable:
+      - appends
+      - asmdecl
+      - assign
+      - atomic
+      - atomicalign
+      - bools
+      - buildtag
+      - cgocall
+      - composites
+      - copylocks
+      - deepequalerrors
+      - defers
+      - directive
+      - errorsas
       - fieldalignment
+      - findcall
+      - framepointer
+      - httpresponse
+      - ifaceassert
+      - loopclosure
+      - lostcancel
+      - nilfunc
+      - nilness
+      - printf
+      - reflectvaluecompare
       - shadow
+      - shift
+      - sigchanyzer
+      - slog
+      - sortslice
+      - stdmethods
+      - stringintconv
+      - structtag
+      - testinggoroutine
+      - tests
+      - unmarshal
+      - unreachable
+      - unsafeptr
+      - unusedresult
+      - unusedwrite
   loggercheck:
     kitlog: false
     klog: false
     require-string-key: true
     no-printf-like: true
   paralleltest:
     ignore-missing: true
+  prealloc:
+    for-loops: true
   revive:
-    enable-all-rules: true
+    # https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md
     rules:
       - name: add-constant
         disabled: true
+      - name: argument-limit
+        arguments: [6]
```

Comment:
I would suggest removing this linter, I understand that most of the time there is no sane need for 6+ args, but what if they are needed, this config will just be changed to new max value